service/controller/v2/arm_templates/main.json

```diff
@@ -347,7 +347,10 @@
           "vmssName": {"value": "[concat(parameters('clusterID'), '-master')]"},
           "vmssVmSize": {"value": "[parameters('masterNodes')[0].vmSize]"},
           "vmssVmCount": {"value": "[length(parameters('masterNodes'))]"},
-          "vmssVmDataDisks": {"value": [{"name": "DockerDisk", "sizeGB": 50}]},
+          "vmssVmDataDisks": {"value": [
+                  {"name": "EtcdDisk", "sizeGB": 100},
```

Comment:
100GB not because we really need 100GB of etcd data, but because Azure has linear dependency between disk size and  performance. So 100GB will be P10 type, which gives 500IOPS and 100MBps, which is fine for etcd (requres 50-500 IOPS sequental)

https://docs.microsoft.com/en-us/azure/virtual-machines/linux/premium-storage

Reply:
Do we know how this behaves with reboots, VM moves and updates?

Reply:
Reboot is fine. VM move and update (vm recreation) will create new disk.

Actually it can be good idea to adress this problem (etcd data preservation) in this PR. Let me check few things.